src/main/views/subscription-search-case-results.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "macros/common-components.njk" import submitButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "macros/common-components.njk" import goBack %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ goBack('/subscription-case-search') }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ header }}</h1>
+  <h3 class="govuk-heading-m">{{ subheader }}</h3>
+
+  <table class="govuk-table govuk-!-width-two-thirds">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">{{ table.header1 }}</th>
+      <th scope="col" class="govuk-table__header govuk-table__header--numeric">{{ table.header2 }}</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__cell--no-line">{{ searchResults.caseNumber }}</th>
+        <td class="govuk-table__cell--no-line govuk-table__cell--numeric">{{ searchResults.caseName }}</td>
+      </tr>
+    </tbody>
+  </table>
+  <form action="subscription-search-case-results" method="post">
```

Comment:
this post action can just be action='pending-subscriptions' it doesn't have to go to subscription-search-case-results' and get redirected to pending-subscriptions